src/Tickets/Commerce/Utils/Value.php

```diff
@@ -75,4 +75,19 @@ public function get_shortcode_price_html() {
 		);
 
 	}
+
+	/**
+	 * Get the display currency.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The display text for this value.
+	 */
+	public function get_currency_display() {
+		if ( $this->get_decimal() == 0 ) {
+			return _x( 'Free', 'No cost', 'event-tickets' );
```

Comment:
I'd add a filter. Post-translation. 
`get_currency()` [already has one](https://github.com/the-events-calendar/event-tickets/blob/master/src/Tribe/Tickets.php#L4253) so I'm not as concerned about that. But we should let folks change this text if they want.

Reply:
Done. Thanks!